VMT/VMT.md

```diff
@@ -0,0 +1,225 @@
+# Vulnerability Management Process
+
+The Kata Containers Vulnerability Management Team (VMT) is responsible for coordinating the progressive disclosure of a vulnerability.
+
+Members of the team are independent and security-minded folks who ensure that vulnerabilities are dealt with in a timely manner and that downstream stakeholders are notified in a coordinated and fair manner.
+Where a member of the team is employed by a downstream stakeholder, the member does not give their employer prior notice of any vulnerabilities.
+In order to reduce the risk of accidental disclosure of vulnerability in the early stages, membership of this team is intentionally limited to a small number of people.
+
+# Acronyms
+
+To aid reading this document it helps to understand some key acronyms:
+
+| Acronym  | Description                                                     |
+| -------- | --------------------------------------------------------------- |
+| CNA      | [CVE Numbering Authority](https://cve.mitre.org/cve/cna.html)   | 
+| CVE      | [Common Vulnerabilities and Exposures](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures) |
+| KCSA     | Kata Containers Security Announcement                           |
+| KCSN     | Kata Containers Security Note                                   |
+| KCST     | Kata Containers Security Team                                   | 
+| PR       | [A GitHub Pull Request](https://help.github.com/articles/about-pull-requests/) |
+| VMT      | The Kata Containers Vulnerability Management Team               |
+
+# Supported versions
+
+> **FIXME** Before we can make statements about which stable versions we fix, we need to define the Kata Containers release and maintenance process.
+
+The VMT coordinates patches fixing vulnerabilities in supported stable branches of Kata Containers, in addition to the master branch (next version under development).
+
+# Process
+
+Each security bug is assigned a VMT *coordinator* (a member from the VMT) that will drive the fixing and disclosure process.
+Here are the steps we follow.
+
+![VMT process workflow](VMT-process.png)
```

Comment:
Diagram comments:

- What is "PTL?" ?
- I think the diagram should show clearly which stages of the process are "private".
- What do the colours mean? I think a key would be useful.
- It might add clarity if we add a "----- time ---->" marker for the x-axis?
- I'd try to avoid italic text for clarity tbh.

Reply:
PTL is an abbreviation for "project team lead" (OpenStack-specific terminology). OpenStack is composed of components maintained by numerous (currently ~70) "project teams" each of whom elect a PTL at the start of every 6-month cycle. The PTL is effectively the chosen representative for the team in all governance matters, though can and often does delegate many of the implied responsibilities to others within their team: https://governance.openstack.org/tc/reference/charter.html#project-team-leads

Reply:
Yep, PTL was in red with a question mark as we do not have them in Kata. I'll nuke that bit - we don't really have an equivalent at present.
I agree, a color key would help - I'll add.
I'll look at adding an x-axis, and contemplate the *italic*s